src/data/multiset/basic.lean

```diff
@@ -933,85 +933,90 @@ multiset.induction_on S (by simp) $
 @[simp] theorem card_join (S) : card (@join α S) = sum (map card S) :=
 multiset.induction_on S (by simp) (by simp)
 
-/-! ### `multiset.bind` -/
+/-!
+### `multiset.bUnion`
+
+The bounded union of an indexed family `f : α → multiset β` of multisets with bound
+`s : multiset α` is the union of all `f a` for `a` in `s` with multiplicity.
```

Comment:
I'm hesitant about using the word `union` here, since the union of multisets has the multiplicity as maximum rather than sum (and `join` uses sum). That said, even the docstring for `join` uses the word union - maybe it would be good to be consistent about this...

Reply:
Interesting, I didn't notice that the union was in terms of maxima.  I guess this means there are two different `bind` operations that could have been defined for multisets, where the one we have now is is terms of multiset sums rather than multiset unions.

Reply:
Ok, I undid the changes to `multiset.bind`.